spec/order_processor_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'order_processor'
+
+describe OrderProcessor do 
+
+	subject { OrderProcessor.new notifier, calculable }
+	let( :notifier ) { double 'notifier', :call => nil }
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside parentheses detected.<br>Tab detected.